app/domain/authentication/installed_authenticators.rb

```diff
@@ -33,20 +33,20 @@ def configured_authenticators
           .push(::Authentication::Common.default_authenticator_name)
       end
 
-      def enabled_authenticators(env)
+      def enabled_authenticators
         # Enabling via environment overrides enabling via CLI
-        env_enabled_authenticators(env) || db_enabled_authenticators
+        env_enabled_authenticators || db_enabled_authenticators
       end
 
-      def enabled_authenticators_str(env)
-        enabled_authenticators(env).join(',')
+      def enabled_authenticators_str
+        enabled_authenticators.join(',')
       end
 
       private
 
-      def env_enabled_authenticators(env)
-        authenticators = env["CONJUR_AUTHENTICATORS"]
-        authenticators.present? ? authenticators.split(',') : nil
+      def env_enabled_authenticators
+        authenticators = Conjur::ConjurConfig.new.authenticators.uniq
```

Comment:
Please use `Rails.application.config.conjur_config.authenticators` within the application code. It gets loaded once when the app starts instead of loading configuration fresh each time ConjurConfig is instantiated.